.npmignore

```diff
@@ -0,0 +1,14 @@
+.idea
+ansible
+config
```

Comment:
For now we might have to include config folder for dynamo or write that module so we can pass in a config settings in a constructor or something like that